src/services/apiService.ts

```diff
@@ -7,7 +7,13 @@ export interface ILoginRequestPayload {
     password: string;
 }
 
-class ApiService {
+export interface IApiService {
+    loginWithCredentials(data: ILoginRequestPayload): Promise<object>;
+    testToken(): Promise<object>;
+    getCurrentUser(): Promise<object>;
```

Comment:
Are there no TS issues with `object` typing? Just asking ;) It usually gives TS errors that property is not found at object type.
Won't you prefer to add some interface for `IUserCredentials`(`access_token` as a field) and `IUser` (`full_name` as a field)?

Reply:
Actually there's no TS errors, but yeah, creating Interfaces is a good idea :) 
I will do this